e2e/tests/nightly/ras-login.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { generate2FACode, signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import DataAccessRequirementsPage, { AccessModule } from 'app/page/data-access/data-access-requirements-page';
+import expect from 'expect';
+import Navigation, { NavLink } from 'app/component/navigation';
+
+/**
+ * NIH Researcher Auth Service (RAS) Login Test
+ *
+ * Important:
+ *   The access test user must be in a state where they are currently failing access renewal
+ *   due to an expired "profile last confirmed" date.
+ *   CircleCI accomplishes this by running the puppeteer-access-test-user-setup task, which executes
+ *   `./project.rb set-access-module-timestamps --profile-user ${ACCESS_TEST_USER} --ras-user ${RAS_TEST_USER}`
+ *   This can also be run locally in workbench/api dir.
+ */
+describe('RAS Test', () => {
+  beforeEach(async () => {
+    await signInWithAccessToken(page, config.RAS_TEST_USER);
+  });
+
+  // note that this test is "destructive" in that it brings the user to a state
+  // where they cannot complete this test again, because they have completed
+  // DAR and are no longer forced into renewal
+
+  test('Sign in login.gov', async () => {
+    // At this time, first page opened after sign in is still the Home page
+    await Navigation.navMenu(page, NavLink.DATA_ACCESS_REQUIREMENTS);
+
+    // Verify configs are not empty strings
+    expect(config.LOGIN_GOV_2FA_SECRET.length).toBeGreaterThan(0);
```

Comment:
please add instructions on how to generate these values for local testing

Reply:
Added instr in test.